docs/userGuide/deployingTheSite.md

```diff
@@ -310,31 +308,14 @@ Now your site will be deployed on Netlify at the given address specified after d
 Additionally, **when contributors make a pull request to your GitHub repo, you can _preview_ the updated site** at the bottom of the pull request by clicking on `details` link in the PR:
 <include src="screenshot.md" boilerplate var-alt="Preview deploy" var-file="netlifyPreview4.png" inline />
 
-{{ icon_info }} Note that when Netlify is set up as explained above, it will use the latest version of MarkBind which may be a later version than the one you use locally. If you want Netlify to use a specific version of Netlify, follow the instructions in the box below:
-
-<box>
+## Relevant Tips & Tricks
 
-##### Setting up Netlify to use a specific version of MarkBind
+<panel header="Configuring Online Deployment platforms to use specific MarkBind version" expand-headerless>
```

Comment:
would 
```suggestion
<panel header="Configuring Online Deployment platforms to use specific MarkBind version" type="seamless" expand-headerless>
```
be more consistent?

Reply:
Yup it looks more consistent. Thanks for the suggestion!